client/spec/content_spec.js

```diff
@@ -12,8 +12,8 @@ describe('Content', function() {
 
     beforeEach(function() {
         openUrl('/#/workspace/content')();
-        workspace.openPersonal();
-        expect(element.all(by.repeater('items._items')).count()).toBe(3);
+        workspace.switchToDesk('PERSONAL');
+        expect(element.all(by.repeater('items._items')).count()).toBe(6);
```

Comment:
fyi you can use `.toBeGreaterThan(x)` so we don't need to update specs when there is a new item added